### PR TITLE
Format table fields 

### PR DESCRIPTION
**Pull request by laughedelic**
> Todas las preguntas que tienen las mismas opciones de respuesta, deben ir en tabla porque de lo contrario se hace complicado y pesado responder.

### CODE REVIEW

**Comment by laughedelic** [comment]
This is a bit more complicated than I expected. Probably [`b3.inputWrapped`](http://adrianhurt.github.io/play-bootstrap/1.1-P25-B3/docs/#your-inputwrapped) is a solution. Use it to wrap each question as a table row with options in separate cells. Also I would remove the labels for options, leaving them only in the header.

**Comment by laughedelic** [comment]
I saw this in the mock that @vdiaz-biomed sent me:

screen shot 2017-04-06 at 4 59 58 pm

So I'm trying to do the same, but I don't see a simple way using [Bootstrap helpers](http://adrianhurt.github.io/play-bootstrap/1.1-P25-B3/docs/#helper-radio). So I'm going to try to generate a similar table and hope not to mess up with those styles..

**Comment by vdiaz-biomed** [comment]
Take into account that the table in the mock has been made using bootstrap table and form elements, plus a bit of styling for the first column of the table.

**Comment by laughedelic** [comment]
Okay, but how did you make it? Writing that html directly (with references to those bootstrap styles/attributes) or using some bootstrap magic?

**Comment by vdiaz-biomed** [comment]
Let's see if this helps you:
Questions grouped in "table mode" follow this html structure:

```
<table class="table table-striped">

<thead>
<tr>
<th class="statement">A statement for the table or group of questions</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
</tr>
</thead>

<tbody>

<!-- here begins a question; this part of the code must be looped through all the questions that are included int he table -->
<tr>
<td>
<label for="field_04" class="control-label">5.- First question in a table</label>
</td>
<td>
<input required="required" disabled="" type="radio" name="field_04" value="1">
</td>
<td>
<input required="required" disabled="" type="radio" name="field_04" value="2">
</td>
<td>
<input required="required" disabled="" type="radio" name="field_04" value="3">
</td>
<td>
<input required="required" disabled="" type="radio" name="field_04" value="4">
</td>
<td>
<input required="required" disabled="" type="radio" name="field_04" value="5">
</td>
</tr>
<!-- here ends a question in the table -->


</tbody>
</table>
```

The things that must be included programmatically in this structure are:
Statement of the table if it exists
The question text
The "name" (specific for each question)

The attributes "value" are 1 to 5 in the example, but their real value will depend on how you will manage data. They should be substituted for the values that should be collected when the form is send.

**Comment by laughedelic** [comment]
Thanks! This is very useful.

**Comment by laughedelic** [comment]
This works one way:
- [x] it shows a table with radio buttons
- [x] you can set values and they get recorded to the DB
- [ ] but it doesn't show the values read from DB (when you open it again)

The last requires some wrangling with [`B3FieldInfo`](https://github.com/adrianhurt/play-bootstrap/blob/master/play25-bootstrap3/module/app/views/b3/package.scala#L33) or something related which carries the value of the field (to set/enable the right radio-button).

**Comment by eparejatobes** [comment]
I bet you can easily do this with [multifields](https://adrianhurt.github.io/play-bootstrap/1.1-P25-B3/multifield/)

**Comment by laughedelic** [comment]
Fixed the last thing. Going to commit some more changes and merge :shipit:

**Comment by laughedelic** [comment]
This is done for all Alumnos pages. I added `templates.radio` and `templates.radiotable` templates, so it's really easy to use and change in other forms.

File: app/models/actions.scala [hunk 1]
```diff
@@ -1015,7 +1015,7 @@ case class Actions(val evals: titanScarph) {
       .set(q17, form.q17)
       .set(q18, form.q18)
       .set(q19, form.q19)
-      .set(q10, form.q20)
+      .set(q20, form.q20)
```
Comment (laughedelic) [inline app/models/actions.scala]: :trollface:👌

**Comment by laughedelic** [comment]
This is done for all Alumnos pages. I added `templates.radio` and `templates.radiotable` templates, so it's really easy to use and change in other forms.

screen shot 2017-04-12 at 16 50 51

**Comment by marina-manrique** [comment]
wow, super cool! @laughedelic 👏 👏

**Comment by epareja** [comment]
Congratulations !!!

**Comment by vdiaz-biomed** [comment]
Rrrrrrocking!! Amazing!!

**Pull request merged**